apps/element-storybook/src/bs4-migration/form-migration.stories.mdx

```diff
@@ -155,76 +153,146 @@ export const Form = () => {
         .string()
         .required('This Field is required.')
         .nullable(),
+      datePicker: yup
+        .mixed<dayjs.Dayjs>()
+        .required('This Field is required.')
+        .nullable(),
       radio: yup
         .string()
         .required('A selection is required'),
-    })),
-  });
-  return (
-    <form onSubmit={handleSubmit(onSubmit)}>
-      <RequiredKey />
-      <TextField
-        label="Textfield"
-        margin="normal"
-        fullWidth
-        {...register('textField')}
-        required
-        error={!!errors.textField}
-        helperText={errors.textField?.message || 'Max 200 characters'}
-      />
-      <Controller
-        control={control}
-        name="selectField"
-        render={({ field: { onChange, value, onBlur } }) => {
-          return (
-            <Autocomplete
-              onChange={(event, value, reason) => {
-                if (reason === 'clear') {
-                  onChange(null);
-                }
-                onChange(value);
-              }}
-              onBlur={onBlur}
-              FieldProps={{
-                margin:"normal",
-                label: 'Select Field',
-                placeholder: 'Value',
-                required: true,
-                error: !!errors.selectField?.message,
-                helperText: errors.selectField?.message,
-              }}
-              options={dropdownOptions}
-              value={value || null}
-            />
-          );
-        }}
-      />
-      <FormControl margin="normal" error={!!errors.radio} required>
-        <FormLabel id="radio-label" component="div">
-          Radio Group
-        </FormLabel>
-        <Controller
-          control={control}
-          name="radio"
-          render={({ field }) => (
-            <RadioGroup aria-labelledby="radio-label" row {...field} aria-required>
-              <FormControlLabel control={<Radio />} value="1" label="Option 1" />
-              <FormControlLabel control={<Radio />} value="2" label="Option 2" />
-            </RadioGroup>
-          )}
-        />
-        {errors.radio ? <FormHelperText id="radio-helper-text">{errors.radio?.message}</FormHelperText> : null }
-      </FormControl>
-      <Button loading={loading} type="submit" variant="contained">
-        Submit
-      </Button>
-    </form>
-  )
-}
+    });\n
```

Comment:
Are these escape characters `\n` needed? They only show up in a few spots

Reply:
it's only when using the <source> blocks it's needed. iirc there were some styling issues with storybook mdx and using the triple tick source block so <source> was required.

Reply:
forgot to answer why it's only in a few spots. It's only needed when you want an extra line of space, otherwise the source block removes any empty lines.